Week3/queries.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+const db = require("./database.js");
+
+
+module.exports = {
+  createUser: async function (userId, user_name, user_last_name) {
+    try {
+      await db.execQuery(`INSERT INTO users(id, user_name, user_last_name) VALUES( ? , ? , ? )`, [userId, user_name, user_last_name]);
```

Comment:
The client usually dont know the id of the next user it will create. That is why it is a good idea to have some AUTO INCREMENT in the user table too. Like I commented before. =)